apps/client/src/components/toasts/toasts.tsx

```diff
@@ -63,12 +82,14 @@ export const Toasts: React.FC<Props> = ({ isLoggedIn }) => {
                 </div>
               </IconContext.Provider>
               <div className={styles.toastsGridLoggedIn}>
-                {upcomingToasts.map((toast, index) => (
+                {pendingToastsWithUsers?.map((toast, index) => (
```

Comment:
Should probably do something in case `pendingToastsWithUsers` is undefined.
Let's discuss this